osu.Game/Overlays/Chat/Tabs/ChannelTabControl.cs

```diff
@@ -88,7 +88,13 @@ public void RemoveChannel(Channel channel)
                 if (nextTabIsSelector && items.Count == 2)
                     SelectTab(selectorTab);
                 else
-                    SwitchTab(nextTabIsSelector ? -1 : 1);
+                {
+                    // If there are any channels in the dropdown (one item more means only selectorTab being there, so we need two)
```

Comment:
i can’t understand this

Reply:
poor wording on my part, sorry.

below, the 'VisibleItems.Count() < Items.Count()' is a check for the dropdown having any items, but only 1 element difference means it's the `selectorTab` in there and not an actual channel. our edge-case only happens if we *do* have channels there, hence the check for one more element.